src/service/vulnerabilities/bola_chatbot_lab.py

```
@@ -292,75 +244,70 @@ async def evaluate_level_2(
         return {
             "level": 2,
             "success": True,
-            "planning_response": planning_text,
+            "planning_response": planning_response,
             "response": response_text,
             "action": action,
             "authenticated_patient": current_patient_id,
         }
-    
+
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)